VERSION.cmake

```diff
@@ -1,6 +1,6 @@
 set (DEFAULT_LIBDNF_MAJOR_VERSION 0)
 set (DEFAULT_LIBDNF_MINOR_VERSION 55)
-set (DEFAULT_LIBDNF_MICRO_VERSION 2)
+set (DEFAULT_LIBDNF_MICRO_VERSION 3)
```

Comment:
[only a suggestion] May be I would recommend to change minor version rather than patch version. But this not an issue.